### PR TITLE
docs: update languages/zh-CN/README.md

### PR DESCRIPTION
**Pull request by JieDing**
- add new translation languages/zh-CN/README.md
- add `README.md` and `RELEASE_NOTES.md` to `languages` sub-dirs of every spec directory
- update languages-spec

Signed-off-by: Jie Ding <jie.ding@linkall.cloud>

#984

### CODE REVIEW

File: languages/zh-CN/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# CloudEvents 规范
```
Comment (duglin) [inline languages/zh-CN/README.md:1]: Should this be under the "cloudevents" dir? Or should there be a "cloudevents" subdir under "languages" ?
I'm leaning more towards "cloudevents/languages/..."

Comment (duglin) [inline languages/zh-CN/README.md:1]: I guess we have a decision to make... should all CE docs (for all languages) be under `/cloudevents` or should we group all docs for each languages under `/languages/LANG` ? 

ie.
```
/README.md
/languages/README.md
/cloudevents/README.md
/cloudevents/languages/zn-cn/README.md
/cloudevents/languages/zn-cn/spec.md
/discovery/README.md
/discovery/languages/zn-cn/README.md
/discovery/languages/zn-cn/spec.md
```
vs
```
/README.md
/cloudevents/README.md
/discovery/README.md
/languages/zh-cn/README.md
/languages/zh-cn/cloudevents/README.md
/languages/zh-cn/cloudevents/spec.md
/languages/zh-cn/discovery/README.md
/languages/zh-cn/discovery/spec.md
```

I'm wondering if our new release process, which seems to be focused on releasing "directories" (sort of), would influence this decision.

Comment (JieDing) [reply languages/zh-CN/README.md:1]: For the first situation, each directory has it's own sub-dir /languages/LANG, while the other grouping all docs for each languages under /languages/LANG.  The difference between these two situations reflects two different specification structure.
The first structure seems to treat each directory as an independent spec.
i.e. 
```
/cloudevents  dir represents the CloudEvents spec
/discovery dir represents the Discovery spec
```
If that's the structure we want to release, then it makes sense that each dir holds its language sub-dir.

The second structure seems to treat all directories as the whole CloudEvents spec.
```
/cloudevents  dir represents the CloudEvents core spec
/discovery dir represents the CloudEvents additional spec about discovery
```
For that structure, it seems better if we group all docs under /languages/LANG.

So it all depends on what kind of spec structure we want to release finally.

Comment (JieDing) [reply languages/zh-CN/README.md:1]: I personally prefer the second one coz I think that the CloudEvents spec should consist of core spec and other related specs.

Comment (JieDing) [reply languages/zh-CN/README.md:1]: Or maybe we can re-org the structure like this:
```
/cloudevents
    /core    -> this core dir holds everything(like SDK.md, spec.md, adapters dir, etc) from the original /cloudevents dir
    /discovery   
    /subscription  
    /schemeregistry  
    /cesql 
    ...
/misc
/community
...
```

Comment (duglin) [reply languages/zh-CN/README.md:1]: Not sure if you were aware of the decision by the group but we decided to have the other spec dirs (eg. "discovery") hold all info for that spec. Basically like a nested git repo, but w/o the overhead of the repo. Then each release would be a 'tag' off of the 'main' branch with the dir being released as part of the tag name. Yes this means we'll be deleting the other branches and just have 'main' going forward. See: https://github.com/cloudevents/spec/pull/968

So, your statement:
> The first structure seems to treat each directory as an independent spec.

is how I view the current direction of the group. But we can wait for other folks to chime in to see if they have a different interpretation.

Comment (JieDing) [reply languages/zh-CN/README.md:1]: I went through the tag-based structure. It does make sense. I noticed it's approved on the 3/18 call. So is this direction settled or not?

Comment (duglin) [reply languages/zh-CN/README.md:1]: yes I just need to find the time to "make it so".

Comment (JieDing) [reply languages/zh-CN/README.md:1]: Cool, I'll update the language spec then to ensure each directory has a "languages" sub-dir.

**Comment by duglin** [comment]
@JieDing let me know when you think each PR Is ready to be considered by the group for merging. I don’t want to do it prematurely

**Comment by JieDing** [comment]
> @JieDing let me know when you think each PR Is ready to be considered by the group for merging. I don’t want to do it prematurely

@duglin I reviewed this PR and thought it's ready for merging.

**Pull request merged**